lib/index.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
-
```

Comment:
why did you remove `'use strict';` everywhere? I know we will build our project in a different way - but how does that relate to source code being strict?